ch.openboard.OpenBoard.json

```diff
@@ -0,0 +1,84 @@
+{
+    "id": "ch.openboard.OpenBoard",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "openboard",
+    "rename-desktop-file": "OpenBoard.desktop",
```

Comment:
Should be renamed upstream. Same for the icon.

Reply:
I'm not really able to do this since I'm not a member of the upstream project. In case the upstream authors want maintain this flatpak, they might want to follow your suggestion.

Thanks for your review btw!

Reply:
You could just open an issue upstream about that and showing them this 
For Appdata: https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html
For Desktop file and icons: https://standards.freedesktop.org/desktop-entry-spec/latest/

Reply:
Fixed in the patches for now. I'll try to upstream them.